designer/server/src/routes/forms/library.js

```diff
@@ -21,19 +22,30 @@ export default [
       handler: async (request, h) => {
         const { auth, query } = request
         const token = auth.credentials.token
+        const { page, perPage, sort } = query
 
-        const { page, perPage } = query
+        let sortBy, order
+        if (sort) {
```

Comment:
Will anything break if sort was split and it didn't contain an underscore?

Just checking

```suggestion
        if (sort?.includes('_')) {
```

Reply:
I've changed it anyhow to use camelCase like the current pagination query params, so `updatedAt` etc. But to answer your question, it's much the same like how it was for invalid pagination query params in that it would show a generic 400:
```
{"statusCode":400,"error":"Bad Request","message":"Invalid request query input"}
```

Reply:
I've added the `failAction` for the invalid query params to throw the correct `Boom()` error.